README.md

```diff
@@ -1,5 +1,9 @@
 # Simorgh
 
+Named Simorgh after the Persian mythological bird. The Simorgh is the amalgam of many birds (and in some accounts other animals) into one. We consider this an apt metaphor for all articles of the BBC in one solution, a clear reference to the international nature of our teams but also to the articles themselves working for international users from the outset. It is also a unique name which is practical and, more superfically, the bird is very pretty. 
+
+![](https://lh6.googleusercontent.com/OsrCnGr2OiqJEbpN7hkutKGqaYpS5M55MKNShymVcCJIk93rlxZLH5rvL3rj8JXphsf86OWh17itiION6WVaO1vU2GWpAOEBnpMSVihwPgUBP7TUirpEJrhkuWZ1xC6j9RK-l2Hg)
```

Comment:
Do we have permission to use this image? I couldn't find the copyright holder when I looked into doing this

Reply:
I think we'd have to contact the gallery http://maryryangallery.com/artists/peter-sis/ to ask permission. There isn't anything about copyright on their website that I can see. We probably shouldn't use the image until we have confirmation.